tests/images/polar_alignment_utils_new.py

```diff
@@ -0,0 +1,60 @@
+from PIL import Image, ImageOps
+
+
+def fix_image_size(max_width, max_height, image):
+    """Resizes images while maintaining the same aspect ratio.
+
+    Args:
+        max_width: Image width
+        max_height: Image height
+        image: Image to be resized
+
+    Returns:
+        The resized image.
+    """
+    width_ratio = max_width / image.size[0]  # resized image ratios
+    height_ratio = max_height / image.size[1]
+
+    new_width = int(width_ratio * image.size[0])  # proportional width, height value calculations
+    new_height = int(height_ratio * image.size[1])
+
+    new_image = image.resize((new_width, new_height))
+    return new_image
+
+
+def grid_display(
+    background_jpg,
+    overlay_png,
+    width=4800,
+    height=2400,
```

Comment:
Usually we only want to scale an image by one size so we can keep the aspect ratio.  We could deal with width and height but it's easier to just ask for a scale. This would require some change to the logic below.  You could alternatively ask for a `max_width` or `max_height` and scale from there but it's probably simplest just to have a simple `scale` factor.

```suggestion
    scale=0.5,
```